routes/project/location/postgres.rb

```diff
@@ -453,6 +453,40 @@ class Clover
           metrics: results
         }
       end
+
+      r.on "config" do
+        r.get true do
```

Comment:
```suggestion
      r.is "config" do
        r.get do
```

`r.on` is used if you want to support additional segments of the request path in this branch. As that does not appear to be the case, it's best to use `r.is`.  With `r.is`, the check for a fully routed path is done once at the branch level, so it doesn't need to be done separately in both the `r.get` and `r.put` calls (that is why the `true` argument is dropped).